pages/linux/arecord.md

```diff
@@ -0,0 +1,11 @@
+# arecord
+
+> Sound recorder for ALSA soundcard driver.
+
+- Record a snippet in "CD" quality:
```

Comment:
I tried this, the only way to stop the recording is `Ctrl+C`?

Reply:
Thanks for your swift review @navarroaxel! 

Yes, that's correct. An alternative is to specify a fixed duration using `-d` or `--duration`. I mostly use it in the dynamic style shown by this snippet, but happy to replace that with the fixed duration one if you think that's generally more useful for people.

Reply:
Added explanation for `Ctrl+C` in b53cdb6406431a583c639dac1b17366d18c460f0

Reply:
Added fixed duration snippet (no `Ctrl+C` required) in de200e42fdb94daaec60902d2c3a28299f38e743